it/src/main/resources/tests/joins/fullOuterEquiJoin.test

```diff
@@ -3,7 +3,6 @@
 
     "backends": {
         "couchbase": "skip",
-        "mimir": "pending",
```

Comment:
@djspiewak is there a reason you do not have `ignoreFieldOrder` here but do in the other tests?